amundsen_application/static/js/components/TableDetail/index.tsx

```diff
@@ -146,7 +145,7 @@ export class TableDetail extends React.Component<TableDetailProps & RouteCompone
       <OverlayTrigger key={fullName} trigger={['hover', 'focus']} placement="top" overlay={popoverHoverFocus}>
         <div style={{display: 'inline-block', marginLeft: '-5px',
           backgroundColor: 'white', borderRadius: '90%'}}>
-          <Avatar name={fullName} size={25} round={true} style={{border: '1px solid white'}}/>
+          <Avatar name={fullName} size={25} round={true} style={{zIndex, position: 'relative'}}/>
```

Comment:
The Avatars here are inheriting the `border: 1px solid white` rule as defined in `NavBar/styles.scss`. I don't think styles like that should leak out into other places that were not originally intended. If the intent is that all Avatars should have the border, then it should be defined at the appropriate level. 

As I mentioned earlier, the NavBar stylesheet could use some cleanup.

Reply:
So I got rid of the inline styling here because it looked like it was unnecessary (.sb-avatar already had a border attribute that was being doubled up by the inline style). With that inline style the border also looked asymmetrical like in [here](https://jira.lyft.net/secure/attachment/224446/image-2019-04-04-16-15-41-093.png)

Reply:
Yes, I completely understand that. I'm just saying `NavBar/styles.scss` could use some TLC.

Reply:
ended up moving the avatar specific global styling into a new file, should hopefully make it a little more clear where global avatar specific styling lives rather than inside of components